src/extension/sdk/utils.ts

```diff
@@ -304,6 +304,7 @@ export class SdkUtils {
 			fuchsiaRoot && path.join(fuchsiaRoot, "dart/tools/sdks", dartPlatformName, "dart-sdk"),
 			firstFlutterMobileProject && flutterSdkPath && path.join(flutterSdkPath, "bin/cache/dart-sdk"),
 			config.sdkPath,
+			workspaceConfig.defaultDartSdk,
```

Comment:
I presume it's ok for this to take priority over anything on `PATH`? This means if a user wants to override this path they'll need to explicitly set `config.sdkPath` and can't just rely on their SDK being picked up from the `PATH` env variable).

Reply:
True, it'd be better to make this last.